tests/unit/test_contrib/test_pydantic/test_integration.py

```diff
@@ -1,26 +1,36 @@
 from typing import Any, Dict, List, Type, Union
 
 import pydantic as pydantic_v2
+import pytest
 from pydantic import v1 as pydantic_v1
+from typing_extensions import Annotated
 
 from litestar import post
 from litestar.contrib.pydantic.pydantic_dto_factory import PydanticDTO
-from litestar.params import Parameter
+from litestar.enums import RequestEncodingType
+from litestar.params import Body, Parameter
 from litestar.status_codes import HTTP_400_BAD_REQUEST
 from litestar.testing import create_test_client
 from tests.unit.test_contrib.test_pydantic.models import PydanticPerson, PydanticV1Person
 
 from . import PydanticVersion
 
 
-def test_pydantic_v1_validation_error_raises_400() -> None:
+@pytest.mark.parametrize(("meta",), [(None,), (Body(media_type=RequestEncodingType.URL_ENCODED),)])
```

Comment:
testing with form encoded data in addition to json sees that we hit the overridden `decode_builtins()` method on the `PydanticDTO`.